src/main/io/gps.c

```diff
@@ -473,6 +543,92 @@ void gpsInitNmea(void)
 #endif // USE_GPS_NMEA
 
 #ifdef USE_GPS_UBLOX
+const uint8_t ubloxUTCStandardConfig_int[5] = {
+        UBLOX_UTC_STANDARD_AUTO,
+        UBLOX_UTC_STANDARD_USNO,
+        UBLOX_UTC_STANDARD_EU,
+        UBLOX_UTC_STANDARD_SU,
+        UBLOX_UTC_STANDARD_NTSC
+};
+
+struct ubloxVersion_s ubloxVersion_map[] = {
+        [UBX_VERSION_UNDEF] = {~0, "UNKNOWN"},
+        [UBX_VERSION_M5] = {0x00040005, "M5"},
+        [UBX_VERSION_M6] = {0x00040007, "M6"},
+        [UBX_VERSION_M7] = {0x00070000, "M7"},
+        [UBX_VERSION_M8] = {0x00080000, "M8"},
+        [UBX_VERSION_M9] = {0x00190000, "M9"},
+        [UBX_VERSION_M10] = {0x000A0000, "M10"},
+};
+
+static uint8_t ubloxAddValSet(ubxMessage_t * tx_buffer, ubxValgetsetBytes_e key, const uint8_t * payload, const uint8_t offset) {
+    size_t len;
+    switch((key >> (8 * 3)) & 0xff) {
+        case 0x10:
+        case 0x20:
+            len = 1;
+            break;
+        case 0x30:
+            len = 2;
+            break;
+        case 0x40:
+            len = 4;
+            break;
+        case 0x50:
+            len = 8;
+            break;
+        default:
+            return 0;
+    }
+    if (offset + 4 + len > MAX_VALSET_SIZE)
+    {
+        return 0;
+    }
+
+    tx_buffer->payload.cfg_valset.cfgData[offset + 0] = (uint8_t)(key >> (8 * 0));
+    tx_buffer->payload.cfg_valset.cfgData[offset + 1] = (uint8_t)(key >> (8 * 1));
+    tx_buffer->payload.cfg_valset.cfgData[offset + 2] = (uint8_t)(key >> (8 * 2));
+    tx_buffer->payload.cfg_valset.cfgData[offset + 3] = (uint8_t)(key >> (8 * 3));
+
+    for (size_t i = 0; i < len; ++i) {
+        tx_buffer->payload.cfg_valset.cfgData[offset + 4 + i] = payload[i];
+    }
+
+    return 4 + len;
+}
+
+static size_t ubloxAddValGet(ubxMessage_t * tx_buffer, ubxValgetsetBytes_e key, size_t offset) {
+    const uint8_t zeroes[8] = {0};
+
+    return ubloxAddValSet(tx_buffer, key, zeroes, offset);
+}
+
+static size_t ubloxValGet(ubxMessage_t * tx_buffer, ubxValgetsetBytes_e key, ubloxValLayer_e layer)
+{
+    tx_buffer->header.preamble1 = PREAMBLE1;
```

Comment:
@freasy could you please respond on this comment, and the next one also?

Reply:
`ubloxValGet( )` is defined but never used anywhere.

There is an odd `UNUSED(ubloxValGet);` in the function `ubloxValSet( )` even though it's not otherwise referenced in that function (or anywhere else).

Perhaps we should remove `ubloxValGet( )` until we need it for requesting key/value info?

Reply:
@ZzyzxTek 
I agree.  I've commented out `ubloxValGet` and `ubloxAddValGet` with a note.